sql/moz-fx-data-shared-prod/firefox_ios_derived/firefox_ios_clients_v1/query.sql

```diff
@@ -17,6 +17,8 @@ WITH first_seen AS (
   WHERE
     submission_date = @submission_date
     AND client_id IS NOT NULL
+    -- filtering out suspicious devices on iOS, for more info see: bug-1846554
+    AND NOT (app_display_version = '107.2' AND submission_date >= '2023-02-01')
```

Comment:
- Shall we filter this in the view so that it's still available for debugging?
- Also, since the traffic went back to normal on May 5, does it make sense to limit the dates instead of excluding them indefinitely?

Reply:
This was my question in the channel but @bani's Answer made sense to me. Adding her answer in the thread below.
"it's random people in the world doing weird stuff that is not going through the app store in a large volume (see this [thread](https://mozilla.slack.com/archives/C041PKJRA7N/p1693338793787679?thread_ts=1680284882.811289&cid=C041PKJRA7N)).
At some point in August when we were working on the funnel I considered time bounding because the original incident was over, and then I noticed a second spike with the same characteristics.
Since this is an app store funnel, a better filter would be anything that isn't the current version (+/- 1 because the release is gradual), but since we may not have high quality data, this filter at least cleans up some garbage that was preventing the iOS funnel from looking like a funnel when we were doing top -> bottom funnel."

Reply:
Ok, in the end doing something similar to what you suggested @lucia-vargas-a with a small variation. Please let me know if you think this is a reasonable approach.

Reply:
Adding a flag to identify them makes sense, data is still available and we can add other scenarios if needed 👍